src/app/containers/StoryPromo/helpers/locator.js

```diff
@@ -0,0 +1,14 @@
+const getLocator = path => {
```

Comment:
Soz, `helpers/locator.js` and `helpers/originCode.js` dont make much sence IMO in storyPromo, can you put them both in `imageSrcHelpers` folder please, gives it context as to what its for